### DISCUSSION TITLE
Fill available space/ Resize adjacent windows at the same time

### DISCUSSION THREAD

**Discussion by unlocomqx** [rxhanson/Rectangle#699] [Ideas]
Hi,

When I have a window snapped to the right with 60% width for example, I would like the next window that I snap to left to take the remaining 40% instead of taking 50%

Another idea (maybe out of the scope of Rectangle) is to resize adjacent/snapped windows together

Please tell me if it's a good idea
Thank you

**Comment by rxhanson** [comment]
The Fill left/right command is available in my paid superset of Rectangle, [Hookshot](https://hookshot.app). It can take a little to get used to: it will only take into account windows that are immediately above or below, and it will resize to 1/2 if repeated or if unable to fit a reasonable size. I primarily use only these commands, with quick throw for pretty much all of my window resizing needs. I do have plans to update the algorithm to take into account more than just the immediate windows, but there's a lot of features I have lined up.

I don't have plans to implement resize adjacent/snapped windows together because resizing the left side of a window as a 3rd party app can be really janky and I don't like it.

**Comment by unlocomqx** [comment]
I already tried Hookshot and expected the fill half to work automatically not behind a shortcut.
I wanted it to work like on Win10, just take whatever space available on the relevant side.
I'll stick to Hookshot for now.

PS: You're programs look really neat, keep it up
